### PR TITLE
README.md file enhanced

### LINKED ISSUE #93: README.md file enhancement
The README.md file can be further enhanced by adding a list of all the open source programs in which this project is a part of. Also there are minor grammatical errors that needs correction.

### PR DESCRIPTION
**Pull request by anirudhsai20**
### Related Issue
- Fixes #93 

### Proposed Changes
- Added list of Open Source programs
- Added a beautiful table for contributors
- Corrected grammatical errors.

### Additional Information
- Any additional information about the change

### Checklist
- [x] Tested
- [x] No Conflicts
- [ ] Change In Code
- [x] Change In Documentation

### Screenshots
Before this PR            | After this PR
:-----------------------: | :-----------------------:
** Original Screenshot ** | ** Updated Screenshot **

sc

### CODE REVIEW

**Comment by Rishabh510** [comment]
This looks good to go @himanshusharma89 . Please approve and add the required tags.

**Review by Rishabh510 — APPROVED** [review]
OK

**Comment by himanshusharma89** [comment]
@anirudhsai20, some modifications are needed:

- [ ] Remove the stretch in the GSSOC image. Use `%` to provide the height and width, so it doesn't look stretched.
- [ ] Remove underscore between Relic Bazaar.
- [ ] Also, remove the Contributors section because I am planning to use a bot that will update the Readme automatically with the new contributors.
- [ ] Squash the commits to one. Issues like this don't require having so many commits. You have to perform your work and finalize it by a commit at the end.

File: README.md [hunk 1]
```diff
@@ -121,10 +121,35 @@ We follow certain guidelines in order to maintain this repository. Please find o
...
 Distributed under the CC0-1.0 License. See [LICENSE](https://github.com/himanshusharma89/relic_bazaar/blob/master/LICENSE) for more information.
 
-<h1 align= "left"><b>The Geeks behind this 🤓</b></h1>
+<h1 align= "left"><b>The Geeks behind RELIC_BAZAAR 🤓</b></h1>
 
 |<img src="https://avatars2.githubusercontent.com/u/44980497?v=4">|<img src="https://avatars2.githubusercontent.com/u/46641503?v=4">|<img src="https://avatars3.githubusercontent.com/u/54989142?v=4">|<img src="https://avatars3.githubusercontent.com/u/40017559?v=4">|
 |----------------------------|--------|----------------------------|--------|
 |<p align="center">[Himanshu Sharma](https://github.com/himanshusharma89)</p>|<p align="center">[Smaranjit Ghose](https://github.com/smaranjitghose)</p>|<p align="center">[Ishan Sharma](https://github.com/ishandeveloper)</p>|<p align="center">[Anush Bhatia](https://github.com/anushbhatia)</p>|
  
+## RELIC_BAZAAR is a part of these open source programs:
```
Comment (smaranjitghose) [inline README.md:130]: Typically when we write informal documentation. Once we have declared our main subject, we need not refer to it by it's name every time.

So example:

If the title of the project is ``Relic Bazar``, so when we say ``The Development Team``, it is implied that we are referring to ``The Development Team of Relic Bazaar``

**Comment by smaranjitghose** [comment]
A general suggestion for READMEs, try to limit the usage of HTML in markdown files as much as possible. This is because they tend to mess up the formatting in smaller screens at times ( Check out the GitHub's official mobile app)

**Comment by himanshusharma89** [comment]
@anirudhsai20 Any updates? If you are going to update this PR, let us know else we'll be closing this.

**Comment by anirudhsai20** [comment]
@himanshusharma89 . I will update this PR within tomorrow. I was busy with my college work. I hope you understand.

**Comment by himanshusharma89** [comment]
> @himanshusharma89 . I will update this PR within tomorrow. I was busy with my college work. I hope you understand.

Okay, take your time.

**Comment by Rishabh510** [comment]
@himanshusharma89 I think we can assign the issue to someone else.

**Comment by himanshusharma89** [comment]
> @himanshusharma89 I think we can assign the issue to someone else.

He already created a PR for this issue. He told me he is having his exams now.

**Comment by anirudhsai20** [comment]
Thank you very much for your patience @himanshusharma89 . Hope I have made the necessary changes. Also, I was not able to fiure out how to sqauash the commits . Actually, I am just a beginner in Open Source so I am not aware of these things. But I will make sure to follow this from now onwards.

**Pull request merged**